src/cloudant/client.py

```diff
@@ -75,6 +76,7 @@ def connect(self):
             self.r_session.auth = (self._user, self._auth_token)
             self.session_login(self._user, self._auth_token)
         self._client_session = self.session()
+        self.r_session.hooks['response'].append(append_response_error_content)
```

Comment:
Please add a comment about what this is doing to help future visitors to the source!